subsys/bluetooth/mesh/provisionee.c

```diff
@@ -657,6 +671,22 @@ static void prov_link_closed(enum prov_bearer_link_status status)
 	}
 
 	bt_mesh_prov_reset_state();
+
```

Comment:
Shouldn't we exit earlier here if provisioning was successful?

Reply:
Agreed, but I think we should re-enable RPR first (if we cancelled it) since this still has some available functionality (reprovisioning)?

Reply:
> I think we should re-enable RPR first (if we cancelled it) since this still has some available functionality (reprovisioning)?

True